src/main/java/org/osc/sdk/controller/DefaultInspectionPort.java

```diff
@@ -26,13 +26,17 @@
 
     private NetworkElement ingressPort;
     private NetworkElement egressPort;
+    private String elementId;
+    private String parentId;
 
     public DefaultInspectionPort() {
     }
 
-    public DefaultInspectionPort(NetworkElement ingressPort, NetworkElement egressPort) {
+    public DefaultInspectionPort(NetworkElement ingressPort, NetworkElement egressPort, String elementId, String parentId) {
```

Comment:
Can we pass Element interface as argument which has both parent and element id, instead of two separate arguments